content/en/logs/guide/identify-and-exclude-noisy-logs.md

```diff
@@ -0,0 +1,110 @@
+4. Select indexes and click on your associated index. This will populate the option to add an exclusion filter.
+5. Select “Add an Exclusion Filter”.
+6. Input the filter name, define the exclusion query by pasting the parsing rule you copied, and set an exclusion percentage.
+
+You can also generate the parsing rule by clicking the **View All** button in the pattern contextual panel and selecting the **</>** option next to the search bar.
+
+Exclusion filters can be disabled at any time by toggling the disable option to the right of the filter. They can also be modified and removed by hovering over the filter and selecting the edit or delete option.
+
+**Note**: If a log matches several exclusion filters, only the first exclusion filter rule is applied. A log is not sampled or excluded multiple times by different exclusion filters.
+
+## 4. Generate metrics to monitor excluded logs
+
+When a log pattern is excluded from Log Explorer, it still is available to view in [Live Tail][4]. However, you can continue to monitor excluded logs over time to generate KPIs from logs at ingest level by creating a new [log-based metric][5]. You can also create an [anomaly detection][6] monitor to notify you of any important changes to an excluded status.
+
+### Add a new log-based metric
+
+To generate a new log-based metric based on your log pattern, go to the Logs Configuration page of your Datadog account and select the [Generate Metrics tab][7], then the **New Metric+** button.
+
+1. Under Define Query, select the **</>** option and input your parsing rule.
+2. Select the field you would like to track: Select `*` to generate a count of all logs matching your query or enter a log attribute (e.g., `@network.bytes_written`) to aggregate a numeric value and create its corresponding count, min, max, sum, and avg aggregated metrics.
+3. Add dimensions to group by: Select log attributes or tag keys to apply to the generated log-based metric to transform them into [tags][8] following the <KEY>:<VALUE> format. Log-based metrics are considered [custom metrics][9]. Avoid grouping by unbounded or extremely high cardinality attributes like timestamps, user IDs, request IDs, or session IDs to avert impacting your billing.
+4. Name your metric: Log-based metric names must follow the [naming metric convention][10].
+
+### Create an anomaly detection monitor
+{{< img src="logs/guide/anomaly-detection-monitor.gif" alt="Anomaly Detection Monitor" responsive="true" style="width:75%;">}}
+
+Anomaly detection is an algorithmic feature that identifies when a metric is behaving differently than it has in the past. Creating an anomaly detection monitor for your excluded logs will alert you of any changes based on your set alert conditions.
+
+1. To set an anomaly detection monitor, go to [Monitors -> New Monitor -> Anomaly][11].
+2. Set the detection method as anomaly detection.
+3. Define your metric by selecting the **</>** option and paste your parsing rule into the search bar.
```

Comment:
This is for a log monitor not an anomaly monitor, the step should be "enter the log-based metric you defined in the previous section"

Reply:
We should maybe describe here what is happening in the gif (split down by service, focus on a given status..)